css/style.css

```diff
@@ -36,44 +29,212 @@ Developer @GA in NYC
+       width: 1024px;
+  }
+
+
+  header{
+      background-color:#033048;
+      color:#f9e42e;
+      width: 100%;
+      margin: auto;
 }
 
-/******************************************
-/* BASE STYLES
-/*******************************************/
-
-body {
-     color: #000;
-     font-size: 12px;
-     line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
+nav{
+  padding-left: 20px;
+  display: inline;
 }
 
-
-/******************************************
-/* LAYOUT
-/*******************************************/
-
-/* Center the container */
-#container {
-  width: 960px;
-  margin: auto;
+h1{
+  font-size: 40px;
+  color:#fff;
+   padding-left: 50px;
+  width: 40%;
 }
 
-header {
-
-}
-
-footer {
-
+  /*Gutter can be done with just padding*/
+
+  ul{
+    list-style-type: none;
+  }
+
+  nav ul>li{
+    float: left;
+  }
+
+  li a{
+    font-size: 22px;
+    font-weight: 300;
+    text-decoration: none;
+    display: block;
+    padding: 0 15px;
+    color: #f9e42e;
+    line-height: 1.5;
+  }
+
+ ul>li:last-child{
+  padding-right: 50px;
+  ;
 }
+  .drop{
+    font-size: 80px;
+    font-weight: 800;
+    padding:0px;
+  }
+
+  .article h3{
+    font-size: 40px;
+    padding-top: 20px;
+    padding-bottom: 20px;
+  }
+
+  .article p{
+    font-weight: 300;
+  }
+
+  .sidebar h4{
+    font-size: 32px;
+    font-weight: 800;
+    max-width: 100%;
+  }
+
+  .sidebar div{
+    font-weight: 300; 
+  }
+
+  .sidebar p{
+    font-size: 22px;
+    font-weight: 300;
+  }
+
+  .sidebar a{
+    font-weight: 600;
+    font-size: 22px;
+    list-style-type: none;
+    margin: 0px 20px 0px;
+    padding: 0px;
+    text-decoration: none;
+    color: #033048;
+    display: block;
+  }
+
+  .readmore{
+    text-align: right;
+    padding-top: 25px;
+    padding-bottom: 25px;
+  }
+
+  .signup{
+    background-color: #f9e42e;
+    padding-bottom: 15px;
+    padding-top: 15px;
+    text-align: center;
+  }
+
+  .signup button{
+    font-size: 12px;
+    font-weight: 800;
+    border: 5px;
+    background-color: #033048;
+    color: white;
+    padding: 10px 30px;
+  }
+
+  .ad{
+    margin-top: 40px;
+  }
+
+
+  .footer{
+    background-color: #121212;
+    padding-top: 30px;
+    padding-bottom: 30px;
+  }
+
+  .footer div{
+    font-size: 24px;
+    font-weight: 800;
+    color:#f9e42e;
+    text-align: center;
+  }
+
+  .twitter{
+    height: 17px;
+    width: 20px;
+    padding-right: 5px;
+  }
+
+  .facebook{
+    height: 17px;
+    width: 10px;
+    padding-left: 5px;
+  }
+
+  .footer img{
+    height: 17px;
+    width: 20px;
+  }
+
+  .footerimages{
+    margin: auto;
+  }
+
+  .footer footer{
+    font-size: 9px;
+    font-weight: 600;
+  }
+
+  .footer div:last-child{
+    font-size: 10px;
+    color: #ffffff;
+  }
+  /******************************************
+  /* LAYOUT
+  /*******************************************/
+
+  /* Center the container */
+  .main{
+    width: 700px;
+    padding: 20px;
+    padding-right: 20px;
```

Comment:
`padding-right: 20px` is redundant since `padding: 20px;` sets padding on all edges